ee/tabby-ui/app/(dashboard)/settings/general/components/network-form.tsx

```diff
@@ -112,15 +107,18 @@ const NetworkForm: React.FC<NetworkFormProps> = ({
 }
 
 export const GeneralNetworkForm = () => {
-  const [{ data }, reexecuteQuery] = useNetworkSetting()
+  const [{ data, stale }, reexecuteQuery] = useNetworkSetting({
+    requestPolicy: 'network-only'
+  })
+
```

Comment:
since the policy is being changed to `network-only`, does it still make sense to use `stale` field? i would assume `data` will be `null` before network request completed?

Reply:
This query is also used elsewhere, and the presence of the default policy 'cache-and-network' used in other places conflicts with 'network-only' used here. Specifying the policy individually is prone to errors, so the plan is to not specify the 'network-only' policy separately, continue to use 'cache-and-network', and combine data and stale to initialize the form.

Reply:
LGTM